src/modules/dashboard/components/Members/Members.tsx

```diff
@@ -0,0 +1,175 @@
+import React, { FC, useState, useMemo, useCallback } from 'react';
+import { defineMessages } from 'react-intl';
+
+import { ROOT_DOMAIN_ID, ColonyRole } from '@colony/colony-js';
+import sortBy from 'lodash/sortBy';
+import MembersList from '~core/MembersList';
+import { SpinnerLoader } from '~core/Preloaders';
+import { useColonySubscribedUsersQuery, AnyUser, Colony } from '~data/index';
+import { useTransformer } from '~utils/hooks';
+import { getAllUserRolesForDomain } from '../../../transformers';
+import UserPermissions from '~admin/Permissions/UserPermissions';
+import Heading from '~core/Heading';
+import { Select, Form } from '~core/Fields';
+
+import styles from './Members.css';
+
+const MSG = defineMessages({
+  loading: {
+    id: 'dashboard.Members.loading',
+    defaultMessage: "Loading Colony's users...",
+  },
+  title: {
+    id: 'dashboard.Members.title',
+    defaultMessage: `Members{domainLabel, select,
+      root {}
+      other {: #{domainLabel}}
+    }`,
+  },
+  labelFilter: {
+    id: 'dashboard.Members.labelFilter',
+    defaultMessage: 'Filter',
+  },
+});
+
+interface Props {
+  colony: Colony;
+}
+
+type Member = AnyUser & {
+  roles: ColonyRole[];
+  directRoles: ColonyRole[];
+};
+
+const displayName = 'dashboard.Members';
+
+const Members = ({ colony }: Props) => {
```

Comment:
There's a case to be made that this component should actually live under `dashboard/ColonyMembers`, but you can leave it as it is for now